trampoline/src/main/java/org/ernest/applications/trampoline/entities/Ecosystem.java

```diff
@@ -1,8 +1,13 @@
 package org.ernest.applications.trampoline.entities;
 
+import lombok.Getter;
+import lombok.Setter;
+
 import java.util.ArrayList;
 import java.util.List;
 
+@Setter
+@Getter
```

Comment:
While Lombok supports putting getter/setter annotations on classes, it would be much more readable to use the explicit `@Data` annotation for both getter/setter. It may be better to reserve these annotations for places where some fields differ in access.

Reply:
Also, this causes better handling of things like `final` on variables.

Reply:
Completely agree.